kinbot/parameters.py

```diff
@@ -194,10 +198,26 @@ def __init__(self, file=None):
             # MESMER specific keywords
             'mesmer_command': 'mesmer',
 
+            # UQ KEYWORDS
+            # UQ off/on (0/1)
+            'uq': 0,                   
+            # Number of mess input files generated for each structure
+            'uq_n': 5,
+            # Max number of mess calculations running at once
+            'uq_max_runs' : 5,
+            # Uncertainty in stationary point energy, +/- 0.5 kcal/mol
+            'stPt_uq': 0.5,
+            # Uncertainty in saddle point (TS) energy, +/- 1.0 kcal/mol
+            'barrier_uq': 1.0,
+            # Uncertainty in positive frequency values, mult/div 1.2 (1.2 = 20% uncertainty)
+            'posFreq_uq': 20,
+            # Uncertainty in negative frequency values, mult/div 1.1 (1.1 = 10% uncertainty)
+            'negFreq_uq': 10,
```

Comment:
I think we should ask the user to explicitly think about multiplicative factors, so I'd prefer to have the input as 1.2 which means 0.8333 to 1.2. 
Most keywords have a structure like a_b, and not aB, so can you change `posFreq` to something else? Maybe a better and consistent name would be (based on the previous things above): `freq_uq` and `imagfreq_uq`.

Reply:
changed in the code